.golangci-kal.yml

```diff
@@ -18,12 +18,14 @@ linters:
               - "conditions" # Ensure conditions have the correct json tags and markers.
               - "conflictingmarkers"
               - "duplicatemarkers" # Ensure there are no exact duplicate markers. for types and fields.
+              #- "forbiddenmarkers" # Ensure that types and fields do not contain any markers that are forbidden.
```

Comment:
I definitely want to surface the linter here to make folks aware that this exists

@JoelSpeed Do you have any suggestions about markers that we should forbid?

Reply:
We need Schemaless in a few cases, but I think we'll have discussions anyway if we want to add this to any new fields. I'll add a rule for UniqueItems